wpeverest/everest-forms
# Tweak - Serial number on entry listing table

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds the serial number column in the entries list.

### How to test the changes in this Pull Request:

1. Go to Everest forms pro.
2. Click on Entries.
3. Check whether the serial number column is coming or not and also verify the hide and show column from the setting.

### Types of changes:


### Other information:


### Changelog entry

>Tweak - Serial number on entry listing table.
